ros/src/waypoint_updater/waypoint_updater.py

```diff
@@ -113,8 +113,9 @@ def __init__(self):
         # Current list of base waypoints from Lane object
         self.base_waypoints = None
 
-        rospy.loginfo('WaypointUpdater Initialized.')
         rospy.spin()
+        rospy.loginfo('WaypointUpdater Initialized.')
+        print "...Waypoints initalized"
```

Comment:
The two lines above will only be called after ROS has been terminated. It does not do what you expect it to do.

Reply:
My bad, it should be before the `spin()`